### ISSUE TITLE
vimr to send buffer into active split

### ISSUE THREAD

**Issue by wzhliang** [qvacua/vimr#527]
Currently, `vimr <file>` opens a new tab. This is different from the default gvim behavior. Is this a designed feature or should it follow gvim?

**Comment by bsia** [comment]
It's also different from the default MacVim behavior which opens the file in a new window.

**Comment by qvacua** [comment]
It's intended. If you want to open the file in a new window, you can use the `-n` or `-s`. See also `vimr --help`. Does this help?

**Comment by wzhliang** [comment]
Not exactly. If there is more than one split in the same window (with Ctrl-W-s for example), `gvim --remote` will send the new buffer to the active split. No new tab, no new window. This is handy when you want to view 2 or more files at the same time. It would be nice.

**Comment by qvacua** [comment]
Ah, ok. Maybe that'd be a better behavior. Let me think about it. 😀

**Comment by evanheller** [comment]
This comes up for me when working within the GUI. Opening files using :e behaves as expected and opens a new file in the current split, but using the File menu, Cmd-O, or the Buffers drawer always defaults to a new tab, and it would be nice to be able to set this behavior to your liking.

**Comment by snoblenet** [comment]
Related: I'd really appreciate a preference setting to open a file in a new vertical split next to the existing VimR window when I double-click a file in the Mac OS X finder and there is an existing window in VimR.

**Comment by bronzehedwick** [comment]
>This comes up for me when working within the GUI. Opening files using :e behaves as expected and opens a new file in the current split, but using the File menu, Cmd-O, or the Buffers drawer always defaults to a new tab, and it would be nice to be able to set this behavior to your liking.

Agree 100% here. It's the only reason I don't use the VimR built in fuzzy finder or file system explorer.
